STATUS: INLINE FAIL SETUPTEARDOWN
MODIFIED: de.strullerbaumann.visualee.ui.graph.boundary.GraphCreatorTest.testBuildJSONNodes
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/ui/graph/boundary/GraphCreatorTest.java
CLEANER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testToString
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.PackageFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/PackageFilterTest.java
ORIGINAL CLEANER SIZE: 6
NEW CLEANER SIZE: 1
ELAPSED TIME: 52.18400001525879
==========================
@@ -66,0 +66,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
